### PR TITLE
Create new annotation fallback tests and cleanup

### PR DESCRIPTION
**Pull request by clapierre**
Moved the annotation / aria-label tests to a new file and test 080 and 090, cleaned up javascript requirements, fixed epubcheck errors due to name spacing. Also added a new test for m: = mathml namespace test 100.

### CODE REVIEW

**Comment by marisademeglio** [comment]
The previous version was 1.1.0. As it sounds like there are breaking changes in this book (new or changed tests), you should increment the MIN value (so: go to `1.2.0` instead of `1.1.1`). See the [versioning rules](https://github.com/daisy/epub-accessibility-tests#about-versioning).

**Comment by clapierre** [comment]
Updated version # to 1.2.0 and date change made to this pull request/branch

File: content/epub30-test-0330/EPUB/Text/epub-image-fallback-mathml-examples.xhtml [hunk 1]
```diff
@@ -111,15 +111,20 @@
...
-        <p class="desc">The test passes if the assistive technology reports the image and its alt text as well as reports presence of the MathML expression and provides functionality to read it.</p>
+        <div class="desc">
+            <p>The test passes if all of the following are true:</p>
+            <ul>
+                <li>The assistive technology reports presence of the MathML expression and provides functionality to read it.</li>
+               <li>The assistive technology reports the image and reads its alt text</li>
+            </ul>
 
-        <p>The test  fails if you only hear the image alt text description and not able to access the offscreen mathML.</p>
+            <p>The test fails if you only hear the image alt text description and not able to access the offscreen mathML.</p>
```
Comment (mattgarrish) [inline content/epub30-test-0330/EPUB/Text/epub-image-fallback-mathml-examples.xhtml]: s/and not able/and are not able/

Comment (clapierre) [reply content/epub30-test-0330/EPUB/Text/epub-image-fallback-mathml-examples.xhtml]: Made these changes.
Thanks Matt

**Pull request merged**